tests/schema.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Don't add a schema, fetch the schema from iopipe/iopipe's repo

Reply:
Ok, fetching now.